tests/storage_test.go

```diff
@@ -324,7 +324,7 @@ var _ = Describe("Storage", func() {
 
 			Context("With a HostDisk defined", func() {
 
-				const hostDiskDir = "/tmp/kubevirt-hostdisks"
+				const hostDiskDir = "/var/run/kubevirt/kubevirt-hostdisks"
```

Comment:
This one may be properly labeled but it has a special purpose and should not be used to place arbitrary files inside it.